erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
+        if (that.room && that.room.p2p){
+            L.Logger.warning('muteAudio is not implemented in p2p streams');
+            callback ('error');
```

Comment:
I think the cost/benefit of implementing something like this in P2P is a lot less appealing. The big win here is in big rooms, as we're saving audio bandwidth. In small rooms/p2p muting the audio in the client side should be sufficient in most cases. I'm not even convinced it's that practical considering the potential delay.

An alternative would be to just to mute the audio track here if we're in p2p.